services/vision/mlvision/ml_model.go

```diff
@@ -96,6 +97,27 @@ func registerMLModelVisionService(
 	for oldName, newName := range params.RemapOutputNames {
 		outNameMap.Store(newName, oldName)
 	}
+	if len(params.BoxOrder) != 0 {
+		if len(params.BoxOrder) != 4 {
+			return nil, errors.Errorf(
+				"attribute xmin_ymin_xmax_ymax_order for model %q must have only 4 entries in the list. Got %v",
+				params.ModelName,
+				params.BoxOrder,
+			)
+		}
+		checkOrder := map[int]bool{0: false, 1: false, 2: false, 3: false}
+		for _, entry := range params.BoxOrder {
+			val, ok := checkOrder[entry]
+			if !ok || val { // if val is true, it means value was repeated
+				return nil, errors.Errorf(
+					"attribute xmin_ymin_xmax_ymax_order for model %q can only have entries 0, 1, 2 and 3, and only one instance of each. Got %v",
+					params.ModelName,
+					params.BoxOrder,
+				)
+			}
+			checkOrder[entry] = true
```

Comment:
Okok I get it. I feel like there's an easier way to check for duplicates but you also had to write an argmax function soo...